packages/@adobe/spectrum-css-temp/components/searchwithin/index.css

```diff
@@ -29,7 +29,7 @@ governing permissions and limitations under the License.
 
 .spectrum-SearchWithin-picker {
   inline-size: auto;
-  min-inline-size: var(--spectrum-global-dimension-size-900);
+  min-inline-size: var(--spectrum-global-dimension-size-1000);
```

Comment:
Bumped up the width a little bit, now looks like this:
![image](https://user-images.githubusercontent.com/9661127/135363703-322a2105-9e64-4829-b8d9-0d1e9e89a315.png)
Playing around a bit with the flex-shrink/grow values to see if I can have the picker grow a bit more

Reply:
You could theoretically have both have grow properties, but by uneven amounts?
flex-grow: 1;
flex-grow: 5;
one would grow 5x faster than the other
is that what you mean?